plugin/src/main/kotlin/io/qalipsis/gradle/cloud/tasks/CloudRunQalipsis.kt

```diff
@@ -0,0 +1,119 @@
+package io.qalipsis.gradle.cloud.tasks
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.qalipsis.gradle.cloud.dsl.CampaignConfigurationSpecification
+import io.qalipsis.gradle.cloud.dsl.CampaignConfigurationSpecificationImpl
+import io.qalipsis.gradle.cloud.model.Campaign
+import io.qalipsis.gradle.cloud.model.ErrorMessage
+import io.qalipsis.gradle.cloud.tasks.Helper.authenticate
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import java.io.IOException
+import javax.inject.Inject
+
+/**
+ * Task in charge of creating new campaigns in the cloud.
+ *
+ * @author Francisca Eze
+ */
+abstract class CloudRunQalipsis @Inject constructor() : DefaultTask() {
+
+    private val mapper = jacksonObjectMapper()
+
+    private lateinit var campaignSpecification: CampaignConfigurationSpecificationImpl
+
+    /**
+     * Uses a DSL-style configuration block to set up a [CampaignConfigurationSpecification].
+     */
+    fun campaign(block: CampaignConfigurationSpecification.() -> Unit) {
```

Comment:
Add the name of the campaign to create as first argument and remove the related setter from CampaignConfigurationSpecification.